executor/prepared.go

```diff
@@ -271,7 +271,7 @@ func CompileExecutePreparedStmt(ctx sessionctx.Context, ID uint32, args ...inter
 	}
 
 	stmt := &ExecStmt{
-		InfoSchema: GetInfoSchema(ctx),
+		InfoSchema: is,
```

Comment:
Is there other place that have unnecessary `GetInfoSchema()`?

Reply:
Not sure, we can do more investigation later.